### DISCUSSION TITLE
turbo caching does not work well with vercel env

### DISCUSSION THREAD

**Discussion by darklight9811** [vercel/turborepo#1208] [Ideas]
Working with vercel's environment variables is being really difficult because there is no easy way to tell turborepo that it shouldn't use a cache. For now I have disabled caching because adding every env variable to globalDependencies is not "scalable".

Since vercel bought turborepo, I tought there could be a way/implementation so both products could go along with each other.

**Comment by anthonyshew** [comment]
Noticing the date on this issue, a lot has changed since then!
